### ISSUE TITLE
Mapillary: Nearest photos doesn't show up.

### ISSUE THREAD

**Issue by dmpr0** [osmandapp/Osmand#8617]
Place http://osmand.net/go.html?lat=55.25094&lon=29.92182&z=17 have Mapillary image, but when open context menu for nearest POI, photos don't show up here, user can view photo only after enabling Mapillary layer and select this point on the map.
[IMG_8728.MP4.zip](https://github.com/osmandapp/Osmand/files/4286953/IMG_8728.MP4.zip)

**Comment by Discostu36** [comment]
Duplicate of #4806, especially [this comment](https://github.com/osmandapp/Osmand/issues/4806#issuecomment-348962042)

**Comment by vshcherb** [comment]
https://github.com/osmandapp/OsmAnd-tools/commit/bdfa4ac78f6421af3aceea746bebb7f7aa77f52e
